src/g3log/logmessage.hpp

```diff
@@ -87,7 +87,7 @@ namespace g3 {
       // Complete access to the raw data in case the helper functions above
       // are not enough.
       //
-      g3::system_time_point _timestamp;
+      g3::high_resolution_time_point _timestamp;
```

Comment:
Feel free to come up with a better name.  What I'm striving for is to hide implementation details for retrieving the time point for g3sinks.